packages/kbn-openapi-generator/docs/USAGE_GUIDE.md

```diff
@@ -0,0 +1,432 @@
+## OpenAPI Generator (`kbn-openapi-generator`) Usage Guide
+
+OpenAPI code generator (`kbn-openapi-generator`) could be used to generate runtime types, documentation, server stub implementations, clients, and much more (later just artifacts) given OpenAPI specification. Usually the main goal is to generate Zod schemas and related TypeScript types and integrate them into routes implementation and the other code. This guide focuses on mainly on that topic.
```

Comment:
`(later just artifacts)` I'm not sure what this means?

Reply:
I changed the into text a bit to improve readability.